### PR TITLE
gui.snuffling: return chopped, filtered traces

### PR DESCRIPTION
**Pull request by HerrMuellerluedenscheid**
Convenience option to apply main control filters in chopper before returning traces.

### CODE REVIEW

File: src/gui/snuffling.py [hunk 1]
```diff
@@ -945,6 +947,18 @@ def rtrue(tr):
             trace_selector_arg = kwargs.pop('trace_selector', rtrue)
             trace_selector_viewer = self.get_viewer_trace_selector(mode)
 
+            if main_bandpass:
+                def apply_filters(traces):
+                    for tr in traces:
+                        if viewer.highpass is not None:
+                            tr.highpass(4, viewer.highpass)
+                        if viewer.lowpass is not None:
+                            tr.highpass(4, viewer.highpass)
```
Comment (miili) [inline src/gui/snuffling.py]: is this `tr.lowpass` and `viewer.lowpass`?

File: src/gui/snuffling.py [hunk 2]
```diff
@@ -945,6 +947,18 @@ def rtrue(tr):
...
+                    return traces
+            else:
+                def apply_filters(traces):
```
Comment (miili) [inline src/gui/snuffling.py]: Maybe merge toss this condition into `apply_filters` above?

File: src/gui/snuffling.py (hunk 1 above)
Comment (HerrMuellerluedenscheid) [reply src/gui/snuffling.py]: whoops

File: src/gui/snuffling.py (hunk 2 above)
Comment (HerrMuellerluedenscheid) [reply src/gui/snuffling.py]: That would produce more if's being visited in case of many small chops.

**Comment by miili** [comment]
Releasing today - Rein damit! :)

**Pull request merged**